appserver/web/weld-integration/pom.xml

```diff
@@ -173,9 +173,7 @@
             <resource>
                 <directory>src/main/resources</directory>
                 <includes>
-                    <include>**/faces-config.xml</include>
-                    <include>**/com.sun.faces.spi.FacesConfigResourceProvider</include>
-                    <include>**/jakarta.enterprise.inject.spi.Extension</include>
+                    <include>**/*</include>
```

Comment:
I think
```xml
          <resource>
              <directory>src/main/resources</directory>
          </resource>
```
(w/o explicit `include`-everything) would do.

Reply:
I'll try that, thanks!